project-base/src/Shopsys/ShopBundle/Model/Category/CurrentCategoryResolver.php

```diff
@@ -33,7 +33,7 @@ public function __construct(
     /**
      * @param \Symfony\Component\HttpFoundation\Request $request
      * @param int $domainId
-     * @return \Shopsys\ShopBundle\Model\Category\Category|null
+     * @return \Shopsys\FrameworkBundle\Model\Category\Category|null
```

Comment:
why? maybe another phpstan extension ?

Reply:
Because the method returns category from framework bundle... maybe we can solve this by using phpstan-symfony extension but im not sure

Reply:
Yes, please ...